.buildkite/pipeline.yml

```diff
@@ -14,6 +14,6 @@ steps:
     artifact_paths:
       - "result-release/bin/*"
   - command:
-      - GITHUB_TOKEN=$(cat ~/niv-bot-token) NIX_PATH=nixpkgs=$(nix eval --raw '(import ./nix/sources.nix).nixpkgs') nix-shell -p curl git gitAndTools.hub --run "curl https://raw.githubusercontent.com/serokell/scratch/587aded2cd1bb624f85b49727272616fdc49f4fc/scripts/release-binary.sh | bash"
+      - GITHUB_TOKEN=$(cat ~/niv-bot-token) NIX_PATH=nixpkgs=$(nix eval --raw '(import ./nix/sources.nix).nixpkgs') nix-shell -p curl git gitAndTools.hub --run "curl https://raw.githubusercontent.com/serokell/scratch/master/scripts/release-binary.sh | bash"
```

Comment:
I'll copy from `scratch` PR:

> I'd rather not do it this way because one day someone may move this script into another location and things will stop working for everybody. Let's use something permanent instead of master.

Such things do happen all the time. Let's push `release-binary` tag and use it here?

Reply:
Done